### PR TITLE
incorporate Changes Value-Only from IDTA-01002-3-0-2

### LINKED ISSUE #407: Do not change Value-Only type for Range: stay with number
**Is your feature request related to a problem? Please describe.**
Up to V3.0 Part 2 the Value-only schema requested

`"RangeValue": {
      "type": "object",
      "properties": {
        "min": {
          "type": "number"
        },
        "max": {
          "type": "number"
        }
      },`

Now the request came up why this is limited to numbers and there was a decision to allow all types for Range as well. However, this does not really make sense from my point of view.

The types for a Property are like follows:

"PropertyValue": {
      "oneOf": [
        {
          "$ref": "#/definitions/StringValue"
        },
        {
          "$ref": "#/definitions/NumberValue"
        },
        {
          "$ref": "#/definitions/BooleanValue"
        }
      ]
    },

However, a Range of Boolean does not make any sense.
A Range of strings may exist but is the order clearly defined?

### PR DESCRIPTION
**Pull request by BirgitBoss**
- new schema (with Range also Boolean and string)
- update changelog
- add xref for Clauses and Annex
- split valueonly.adoc: 
   extract schema into separate file value-only_json-schema.adoc
- formatting: substitute $\{ with ${

This PR also contain the fix #407 value only range 
that was incorporated into IDTA-01002-3-0-2

### CODE REVIEW

**Review by sebbader-sap — APPROVED** [review]

**Comment by sebbader-sap** [comment]
@BirgitBoss content-wise the changes are fine, please just have a look on how I have resolved the merge conflicts. I am not sure if I did it completely right (e.g. is it `$\{...}` or  `${...}`?).

**Comment by BirgitBoss** [comment]
` ${...}` is fine

File: documentation/IDTA-01001/modules/ROOT/pages/includes/valueonly.adoc [hunk 1]
```diff
@@ -45,6 +47,7 @@ A leaf submodel element follows the rules for the different submodel elements co
 If it is not a leaf element, the serialization rules must be transitively followed until the value is a leaf submodel element.
 * For each submodel element within the submodel, the submodel collection or submodel list:
 
+
 ** xref:spec-metamodel/submodel-elements.adoc#Property[Property]  is serialized as $\{Property/idShort}: $\{Property/value} where $\{Property/value} is the JSON serialization of the respective property’s value in accordance with the data type to value mapping (see table after this section).
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/includes/valueonly.adoc:51]:
```suggestion
** xref:spec-metamodel/submodel-elements.adoc#Property[Property]  is serialized as ${Property/idShort}: ${Property/value} where ${Property/value} is the JSON serialization of the respective property’s value in accordance with the data type to value mapping (see table after this section).
```

File: documentation/IDTA-01001/modules/ROOT/pages/includes/valueonly.adoc [hunk 2]
```diff
@@ -45,7 +47,8 @@ A leaf submodel element follows the rules for the different submodel elements co
 If it is not a leaf element, the serialization rules must be transitively followed until the value is a leaf submodel element.
 * For each submodel element within the submodel, the submodel collection or submodel list:
 
-** xref:spec-metamodel/submodel-elements.adoc#Property[Property]  is serialized as $\{Property/idShort}: $\{Property/value} where $\{Property/value} is the JSON serialization of the respective property’s value in accordance with the data type to value mapping (see table after this section).
+
+** xref:spec-metamodel/submodel-elements.adoc#Property[Property]  is serialized as ${Property/idShort}: ${Property/value} where ${Property/value} is the JSON serialization of the respective property’s value in accordance with the data type to value mapping (see table after this section).
 ** xref:spec-metamodel/submodel-elements.adoc#MultiLanguageProperty[MultiLanguageProperty]  is serialized as named JSON object with $\{MultiLanguageProperty/idShort} as the name of the containing JSON property.
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/includes/valueonly.adoc:52]:
```suggestion
** xref:spec-metamodel/submodel-elements.adoc#MultiLanguageProperty[MultiLanguageProperty]  is serialized as named JSON object with ${MultiLanguageProperty/idShort} as the name of the containing JSON property.
```

**Comment by BirgitBoss** [comment]
Check after whether there are $\{ in text

**Pull request merged**